CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 
-## VNext
+
+## vNext 
+- Updated URL validation
```

Comment:
Let's keep the right order here. This update should be in the second line as far as `Added assert` is already in the `dev`.

Reply:
Updated